updating/updating-cluster-within-minor.adoc

```diff
@@ -18,11 +18,11 @@ Use the web console or `oc adm upgrade channel _<channel>_` to change the update
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
 +
-{product-title} 4.9 requires an upgrade from etcd version 3.4 to 3.5. If the etcd Operator halts the upgrade, an alert is triggered. To clear this alert, ensure that you have a current etcd backup and restart the upgrade using the `--force` flag.
+{product-title} 4.9 requires an upgrade from etcd version 3.4 to 3.5. If the etcd Operator halts the upgrade, an alert is triggered. To clear this alert, cancel the update with the following command:
```

Comment:
What kind of alert is out on it? ClusterNotUpgradeable?

Reply:
Yeah, I'd expect `ClusterNotUpgradeable`.  Might get something else out of the etcd operator with a shorter `for`, but if folks attempt to test this, looking for a pending `ClusterNotUpgradeable` while etcd was happily snapshotting would be fairly convincing evidence that a hung etcd snapshot would cause that to eventually fire.

Reply:
Thanks for the confirmation. Then it looks good to me.